### PR TITLE
Adding Move programming language book to the list of books 

### PR DESCRIPTION
**Pull request by baja-kralj**
## What does this PR do?
Add resource

## For resources
Move is a safe and reliable language originally created by [Diem](https://diem.com/). You can read language white paper on [Diem developers website](https://diem-developers-components.netlify.app/papers/diem-move-a-language-with-programmable-resources/2020-05-26.pdf).

This book is a collection of all the information about Move language created by the team at Mysten Labs.

### Why is this valuable (or not)?
The Move programming language is used currently as a Smart Contract language on Aptos and Sui blockchains, and is in great demand

### How do we know it's really free?
The creators of the Move book website use GNU general public v3.0 license

### CODE REVIEW

File: books/free-programming-books-langs.md [hunk 1]
```diff
@@ -1640,6 +1641,11 @@ Books on general-purpose programming that don't focus on a specific language are
 * [The Little MongoDB Book](http://openmymind.net/2011/3/28/The-Little-MongoDB-Book/)
 
 
+### Move
+
+*[The Move Book](https://move-book.com/index.html)
```
Comment (Thenlie) [inline books/free-programming-books-langs.md:1646]:
```suggestion
* [The Move Book](https://move-book.com/index.html) - Damir Shamanaev
```

**Review by Thenlie — CHANGES_REQUESTED** [review]

**Comment by baja-kralj** [comment]
I committed the changes @Thenlie

**Comment by Thenlie** [comment]
@baja-kralj Formatting looks good to me. Just a question of if we want to add this new language which I will defer to the other reviewers.

**Comment by eshellman** [comment]
In decide where to put a resource that we want to list, such as this one, we have 3 considerations.
1. where are people who should be interested in the resource most likely to discover it?
2. Where will someone most likely to try to put it 6 months from now and discover it to be already listed
3. Will we be stuck with an "orphan category

Answers:
1. in the "blockchain" subject area. 
2. in a new  "Move" language area. (because new users ignore the advise to use the search facility)
3. possibly

My preference is to put it in "blockchain" (#1), then relocate it when more resources appear. A good argument can be made for #2, but a clincher would be if there's already content related to Move in the blockchain area.

Please consider the 3 criteria, and we'll merge based on your considered recommendation.

**Comment by baja-kralj** [comment]
I only considered "by programming language" list, because it is a book about the programming language, for subject it should go in "blockchain", but I don't see Solidity books referenced in "BY SUBJECT" list. @eshellman

**Comment by eshellman** [comment]
Solidity has 2 books and 6 courses. What is your considered recommendation?

**Pull request closed without merging**